python/utils/worker.py

```diff
@@ -122,14 +124,30 @@ def run(self):
             ),
         ]
 
+        # Get the API gateway endpoint
+        match (self.config.grpc_data_stream_network):
+            case "mainnet":
+                grpc_data_stream_endpoint = "grpc.mainnet.aptoslabs.com:443"
+            case "testnet":
+                grpc_data_stream_endpoint = "grpc.testnet.aptoslabs.com:443"
+            case "devnet":
+                grpc_data_stream_endpoint = "grpc.devnet.aptoslabs.com:443"
```

Comment:
why are these hardcoded in code instead of in the config?

Reply:
oh i originally moved them over to code so that it's easier to just specify mainnet/testnet/devnet in config. but with the api gateway it's already pretty self explanatory. i'll move it back.